### PR TITLE
AT_04.23.12 | Verify "Notes/Remark" label is present and visible

### PR DESCRIPTION
**Pull request by Forostinko**
https://trello.com/c/iSusEofv/816-at042312-verify-notes-remark-label-is-present-and-visible

### CODE REVIEW

File: cypress/pageObjects/CreateBookingPage.js [hunk 1]
```diff
@@ -77,6 +77,7 @@ class CreateBookingPage {
     getExtraFareTypeDrop = () => cy.get('.fare-type-box .select2-selection__rendered');
     getDropdownPassengerDefault = () => cy.get('.passenger-wrapper option[value="1"]');
     getRemovePassengerBtns = () => cy.get('.passenger-row .btn-remove-passenger');
+    getNotesRemarkLabel = () => cy.get('.notes-row:nth-child(3) > label:nth-child(1)');
```
Comment (elviraKh1) [inline cypress/pageObjects/CreateBookingPage.js:80]: you could use a shorter selector, just  ".notes-row > label"

**Review by elviraKh1 — APPROVED** [review]

File: cypress/e2e/testUiAndFunction/createBookingPage/US_04.23_PassengersDetailsDefaultUI.cy.js [hunk 2]
```diff
@@ -73,4 +73,19 @@ describe('US_04.23 | Passengers details default UI', () => {
...
+            .should('be.visible');
+    });
+
+    it.only('AT_04.23.13 | Verify Fare type has “Adult” option displayed by default.', function ()  {
+        createBookingPage
+            .getExtraFareTypeDrop()
+            .should('have.text', this.createBookingPage.ExtraFareTypeDrop);
+            // .getFareTypeLabel()
+            // .should('be.visible')
+            // .and('include.text', this.createBookingPage.dropdowns.fareType.labelName);
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/createBookingPage/US_04.23_PassengersDetailsDefaultUI.cy.js:89]: comments should be removed

**Review by elviraKh1 — APPROVED** [review]

File: cypress/e2e/testUiAndFunction/createBookingPage/US_04.23_PassengersDetailsDefaultUI.cy.js [hunk 3]
```diff
@@ -73,4 +73,19 @@ describe('US_04.23 | Passengers details default UI', () => {
...
         createBookingPage.getDropdownPassengerDefault().should('have.text', this.createBookingPage.passengerDefault)
     });
+
+    i('AT_04.23.12 | Verify "Notes/Remark" label is present and visible.', function ()  {
+        createBookingPage
+            .getNotesRemarkLabel()
+            .should('be.visible');
+    });
+
+    it.only('AT_04.23.13 | Verify Fare type has “Adult” option displayed by default.', function ()  {
```
Comment (elviraKh1) [inline cypress/e2e/testUiAndFunction/createBookingPage/US_04.23_PassengersDetailsDefaultUI.cy.js:83]: it only shoul be removed

**Review by elviraKh1 — CHANGES_REQUESTED** [review]

File: cypress/e2e/testUiAndFunction/createBookingPage/US_04.23_PassengersDetailsDefaultUI.cy.js (hunk 3 above)
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/createBookingPage/US_04.23_PassengersDetailsDefaultUI.cy.js:83]: Please delete only

File: cypress/pageObjects/CreateBookingPage.js [hunk 4]
```diff
@@ -77,6 +77,8 @@ class CreateBookingPage {
     getExtraFareTypeDrop = () => cy.get('.fare-type-box .select2-selection__rendered');
     getDropdownPassengerDefault = () => cy.get('.passenger-wrapper option[value="1"]');
     getRemovePassengerBtns = () => cy.get('.passenger-row .btn-remove-passenger');
+    getNotesRemarkLabel = () => cy.get('.notes-row > label');
+    getFareTypeDrop = () => cy.get('#select2-passenger-fare-ur-container');
```
Comment (Rom-T) [inline cypress/pageObjects/CreateBookingPage.js:81]: the same locator on line 60

**Review by elviraKh1 — DISMISSED** [review]

File: cypress/e2e/testUiAndFunction/createBookingPage/US_04.23_PassengersDetailsDefaultUI.cy.js [hunk 5]
```diff
@@ -73,4 +73,10 @@ describe('US_04.23 | Passengers details default UI', () => {
         
         createBookingPage.getDropdownPassengerDefault().should('have.text', this.createBookingPage.passengerDefault)
     });
+
+    i('AT_04.23.12 | Verify "Notes/Remark" label is present and visible.', function ()  {
```
Comment (elviraKh1) [inline cypress/e2e/testUiAndFunction/createBookingPage/US_04.23_PassengersDetailsDefaultUI.cy.js:77]: i -> it

**Review by Rom-T — APPROVED** [review]

**Review by EkaterinaRomankova — APPROVED** [review]

**Pull request merged**